pom.xml

```diff
@@ -29,13 +29,25 @@
       <!-- I used to work for MITRE for many years but I don't anymore. -->
       <!--<organization>MITRE</organization>-->
     </developer>
+    <developer>
```

Comment:
I think there's a `<role>contributor</role>` or something like that which is supported.  Is the presence of your name here a big deal here for you?  I can think of a contributor or two that contributed substantially more than this PR (which doesn't mean I'm not grateful for all contributions!).  Maybe me finally creating a CHANGES.txt would be a good thing to appease the desire to get credit beyond git history.  It'd list names.